classes/EventRegistry.php

```diff
@@ -29,25 +29,36 @@ public function registerFormFieldReplacements($widget)
         // Replace with ML Controls for translatable attributes
         $this->registerModelTranslation($widget);
 
-        // Handle URL translations
-        $this->registerPageUrlTranslation($widget);
+        // Handle Page translations
+        $this->registerPageTranslation($widget);
     }
 
     //
-    // Translate URLs
+    // Translate Page
     //
 
-    public function registerPageUrlTranslation($widget)
+    public function registerPageTranslation($widget)
     {
         if (!$model = $widget->model) {
             return;
         }
 
-        if (
-            $model instanceof Page &&
-            isset($widget->fields['settings[url]'])
-        ) {
-            $widget->fields['settings[url]']['type'] = 'mltext';
+        if ($model instanceof Page)
+        {
+            if (isset($widget->fields['settings[title]']))
```

Comment:
Is there a cleaner way to do this? Why can't we use https://github.com/rainlab/translate-plugin/blob/master/classes/EventRegistry.php#L107?